crit/utils.go

```diff
@@ -246,3 +258,129 @@ func (ps *PsTree) FindPs(pid uint32) *PsTree {
 	}
 	return nil
 }
+
+// Helper to convert slice of uint32 into IP address string
+func processIP(parts []uint32) string {
```

Comment:
Would it be possible to add unit tests for this function?

Reply:
Yes that makes sense. I'll add it, and look into adding it for any other utility functions too if possible.